### DISCUSSION TITLE
Getting parallax to work in tmux and Zellij

### DISCUSSION THREAD

**Discussion by sommerper** [wezterm/wezterm#4586] [Q&A]
### What Operating System(s) are you running on?

Linux X11

### Which Wayland compositor or X11 Window manager(s) are you using?

DE: GNOME 42.5
WM: Mutter

### WezTerm version

20230712-072601-f4abf8fd

### Ask your question!

I really like the effect that parallaxing gives when scrolling but unfortunately it doesn't work in tmux and Zellij. 

I'm wondering if it's possible to get this to work in those two, maybe be manually updating the parallax state or something.

I'm not sure if tmux and Zellij have scrolling hooks that one could listen to.

Any solution or ideas are welcome :-)

**Comment by wez** [comment]
If you can configure them to not use the alternate screen, and run in the main screen, you might be able to get this to work

**Comment by sommerper** [comment]
I couldn't figure out a solution - but it's probably also not that easy to fix. Thanks for the reply anyway!
